packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingToolMarks/DrawingToolMarks.spec.js

```diff
@@ -5,6 +5,8 @@ import { Mark } from '@plugins/drawingTools/components'
 import SVGContext from '@plugins/drawingTools/shared/SVGContext'
 import DrawingToolMarks from './DrawingToolMarks'
 
+// recommend converting this from enzyme to RTL and testing filtering for current frame
```

Comment:
This would be a great opportunity to utilize storybook and RTL for various test cases.